plugins/com.google.cloud.tools.eclipse.bugreport/plugin.properties

```diff
@@ -0,0 +1 @@
+bugReportMenu=Report Cloud Tools for Eclipse bug...
```

Comment:
This doesn't read quite right — the _bug_ is too far away.

_Report bug about Cloud Tools for Eclipse…_?

Or since it's in our GCP menu, could it just be _Report Bug…_?